cmd/crc/cmd/status_test.go

```diff
@@ -170,3 +172,37 @@ func TestJsonStatusWithError(t *testing.T) {
 `
 	assert.Equal(t, expected, out.String())
 }
+
+func TestStatusWithMemoryPodman(t *testing.T) {
+	cacheDir, err := ioutil.TempDir("", "cache")
+	require.NoError(t, err)
+	defer os.RemoveAll(cacheDir)
+
+	client := mocks.NewClient(t)
+	require.NoError(t, ioutil.WriteFile(filepath.Join(cacheDir, "crc.qcow2"), make([]byte, 10000), 0600))
+
+	client.On("Status").Return(apiClient.ClusterStatusResult{
+		CrcStatus:        string(state.Running),
+		OpenshiftStatus:  string(types.OpenshiftRunning),
+		OpenshiftVersion: "4.5.1",
+		DiskUse:          10_000_000_000,
+		DiskSize:         20_000_000_000,
+		RAMSize:          1_000_000,
+		RAMUse:           900_000,
+		Preset:           preset.OpenShift,
```

Comment:
nit: is this should be `preset.Podman`?

Reply:
ohh, i asked simply because the function was named `TestStatusWithMemoryPodman` but the preset was openshift